src/reducers/app.js

```diff
@@ -0,0 +1,62 @@
+import { Record, List } from 'immutable';
+import enUS from 'antd/lib/locale-provider/en_US';
+import frFR from 'antd/lib/locale-provider/fr_FR';
+import getTranslations, { getAvailableLocale } from '../utils/locales';
+
+const SET_LOCALE = 'faucet/locale/set';
+
+export const App = new Record({
+    locale: '',
+    locales: {
```

Comment:
do the available locales and list of steps belong in state? will they ever change during runtime?

Reply:
Steps I would keep in the state because we derive a lot of app state from that, e.g. current step number, next step, and previous step, one could also imagine a different set of steps for different signup flows - invitees for example.

Will remove locales.